src/status_im2/contexts/wallet/common/temp.cljs

```diff
@@ -73,3 +64,17 @@
    :account-name        "Account 1"
    :account             :default
    :customization-color :blue})
+
+(defn keypair-string
+  [full-name]
+  (let [first-name (first (string/split full-name #" "))]
+    (i18n/label :t/keypair-title {:name first-name})))
+
+(defn create-account-state
+  [name]
+  [{:title        (keypair-string name)
+    :button-props {:title (i18n/label :t/edit)}
+    :left-icon    :i/placeholder}
```

Comment:
icon should actually be placeholder? or this is something needs updating?

Reply:
Settings list component needs to be updated https://github.com/status-im/status-mobile/issues/17124